template/pyproject.toml.jinja

```diff
@@ -76,22 +76,22 @@ build-backend = "setuptools.build_meta"
 include-package-data = true
 
 [tool.setuptools.dynamic]
-version = {attr = "{{cookiecutter.module_name}}.__init__.__version__"}
+version = {attr = "{{module_name}}.__init__.__version__"}
```

Comment:
Based on this issue, these lines should be removed when setuptools_scm is selected.

https://github.com/napari/cookiecutter-napari-plugin/issues/185